### PR TITLE
PR: Add various camera spectral sensitivities.

### PR DESCRIPTION
**Pull request by KelSolaar**
References #255.

### CODE REVIEW

File: colour/characterisation/cameras.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        """
+        Setter for **self.x** attribute.
+
+        Parameters
+        ----------
+        value : object
+            Attribute value.
+        """
+
+        raise AttributeError('"{0}" attribute is read only!'.format('red'))
```
Comment (KelSolaar) [inline colour/characterisation/cameras.py]: I'm not sure to understand, the setter is above.

```
    @property
    def red(self):
        """
        Property for **self.x** attribute.

        Returns
        -------
        SpectralPowerDistribution
            self.x

        Warning
        -------
        :attr:`RGB_SpectralSensitivities.red` is read only.
        """

        return self.x

    @red.setter
    def red(self, value):
        """
        Setter for **self.x** attribute.

        Parameters
        ----------
        value : object
            Attribute value.
        """

        raise AttributeError('"{0}" attribute is read only!'.format('red'))
```

Comment (MichaelMauderer) [inline colour/characterisation/cameras.py]: Maybe I'm confused, but doesn't the `@property` only create the getter? If no setter is specified trying to set the property you should see an `AttributeError: can't set attribute`.

Comment (KelSolaar) [inline colour/characterisation/cameras.py]: The setter is defined as per: `@red.setter` and will raise the read only exception: `"red" attribute is read only!`

I suspect I'm missing something here :)

**Comment by MichaelMauderer** [comment]
I think I'm just saying you'd also get an attribute error without implementing the setters explicitly.

Not necessarily anything wrong with doing it, but might not be necessary.

________________________________

**Comment by KelSolaar** [comment]
Ohh right I see! Yeah if it helps trimming the fat we could go for it. That *read only* stuff is re-occurring often through the API. We can probably address that in a later overall pass?

**Comment by MichaelMauderer** [comment]
Ah, hand;t noticed it anywhere else before. But sure, lets look at this some other time then.
Everything else seems fine.

**Pull request merged**